### PR TITLE
add API3 under oracles.mdx

### PR DESCRIPTION
**Pull request by vanshwassan**
## What is the purpose of the change?

Added [API3](https://api3.org/) under `dev-ecosystem-providers/oracles.mdx` 

## Describe the changes to the documentation

Basic developer docs and references for using API3's price feeds on Sei and accessing the [API3 Market](https://market.api3.org/sei). 

## Notes

N.A.

### CODE REVIEW

**Comment by cordt-sei** [comment]
Please add to the bottom of the list to keep the list in chronological order.

File: pages/dev-ecosystem-providers/oracles.mdx [hunk 1]
```diff
@@ -3,4 +3,18 @@
 Oracles provide external data to smart contracts, enabling more dynamic and responsive applications. Notable oracles for Sei include:
 
 ## **Pyth**
-   - [Pyth Documentation](https://docs.pyth.network/home)
+
+- [Pyth Documentation](https://docs.pyth.network/home)
+
+## **API3**
+
+The [API3 Market](https://market.api3.org/) provides access to 200+ price feeds on [Sei Mainnet](https://market.api3.org/sei) and [Testnet](https://market.api3.org/sei-testnet). The price feeds operate as a native push oracle and can be activated instantly via the Market UI.
```
Comment (ghost) [inline pages/dev-ecosystem-providers/oracles.mdx:11]:
```suggestion
The [API3 Market](https://market.api3.org/) offers access to over 200 price feeds on both Sei [Mainnet](https://market.api3.org/sei) and [Testnet](https://market.api3.org/sei-testnet). These price feeds function as a native push oracle and can be activated instantly via the Market UI.
```

File: pages/dev-ecosystem-providers/oracles.mdx [hunk 2]
```diff
@@ -3,4 +3,18 @@
...
+
+The price feeds are delivered by an aggregate of [first-party oracles](https://docs.api3.org/explore/airnode/why-first-party-oracles.html) using signed data and support [OEV recapture](https://docs.api3.org/explore/introduction/oracle-extractable-value.html).
+
+Unlike traditional data feeds, reading [API3 price feeds](https://docs.api3.org/guides/dapis/) enables dApps to auction off the right to update the price feeds to searcher bots which facilitates more efficient liquidation processes for users and LPs of DeFi money markets. The OEV recaptured is returned to the dApp.
```
Comment (ghost) [inline pages/dev-ecosystem-providers/oracles.mdx:15]:
```suggestion
Unlike traditional data feeds, accessing [API3 price feeds](https://docs.api3.org/guides/dapis/) allows dApps to auction off the right to update the feeds to searcher bots, improving the efficiency of liquidation processes for users and LPs in DeFi money markets. The recaptured OEV is returned to the dApp.
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: pages/dev-ecosystem-providers/oracles.mdx (hunk 2 above)
Comment (cordt-sei) [reply pages/dev-ecosystem-providers/oracles.mdx:15]: I think `allows dApps to auction` should be `allow dApps to auction`

**Comment by cordt-sei** [comment]
Hey @vanshwassan can you check the conflicts and requested changes?

**Comment by ghost** [comment]
May we also avoid adding the yarn.lock file

**Comment by cordt-sei** [comment]
Seems abandoned

**Pull request closed without merging**